css/css-view-transitions/transition-in-hidden-page.html

```diff
@@ -22,50 +22,49 @@
       assert_true(document.hidden);
       const transition = document.startViewTransition();
       await wsc.restore();
-      const [readyResult] = await Promise.allSettled([transition.ready]);
-      assert_equals(readyResult.status, "rejected");
+      await promise_rejects_dom(t, "InvalidStateError", transition.ready);
     }, "A view transition should be immediately skipped if started when document is hidden");
 
     promise_test(async t => {
       assert_implements(document.startViewTransition, "Missing document.startViewTransition");
 
       const wsc = window_state_context(t);
       const transition = document.startViewTransition(async () => {
-      await wsc.minimize();
-    });
+        await wsc.minimize();
```

Comment:
I don't understand all these indentation changes. They certainly look wrong and are not explained by the commit message.

Reply:
The `await` here belongs to the anonymous `document.startViewTransition` update callback, not the promise_test, so it makes sense to indent.

Reply:
Oh indeed. Thanks!